koku/masu/util/aws/common.py

```diff
@@ -796,7 +796,7 @@ def clear_s3_files(csv_s3_path, provider_uuid, start_date, metadata_key, metadat
             try:
                 existing_object = obj_summary.Object()
                 metadata_value = existing_object.metadata.get(metadata_key)
-                if metadata_value != metadata_value_check:
+                if str(metadata_value) != str(metadata_value_check):
```

Comment:
What is the type cast for?

Reply:
We had an issue where we were comparing an int to str. Just trying to prevent that.